pages/integrations/sso.md

```diff
@@ -76,7 +76,7 @@ You can configure the session duration to any timeout between 6 hours and 8,760
 ## Frequently asked questions
 
 ### Can some people in the organization use SSO and others not?
-Yes, team maintainers can select whether a user is 'required' to use SSO or whether it is 'optional'. This setting can be found in the team settings.
+Yes, team maintainers can select whether a user is 'required' to use SSO or whether it is 'optional'. This setting can be found in the user settings within the organization's settings page.
```

Comment:
I think this could be tightened a bit. Thinking:

```suggestion
Yes, team maintainers can select whether a user is 'required' to use SSO or whether it is 'optional'. You can find this setting in the organization's user settings.
```

What do you think?

Reply:
I thought you couldn't link to the page because you need the org in the URL, but I've just realised you can:

```suggestion
Yes, team maintainers can select whether a user is 'required' to use SSO or whether it is 'optional'. You can find this setting in the [organization's user settings](https://buildkite.com/organizations/~/users).
```